Example/Controllers/SwiftUI/Features/Settings/Features/UniversalBridgeView.swift

```diff
@@ -0,0 +1,41 @@
+import SwiftUI
+import MiniApp
+
+struct UniversalBridgeView: View {
+    @ObservedObject var viewModel: MiniAppSettingsViewModel
+
+    @State var textToSend: String = "{\\\"data\\\":\\\"This is a sample json information from host app.\\\"}"
+
+    var body: some View {
+        VStack {
+            TextEditor(text: $textToSend)
+                .lineLimit(15)
+                .frame(width: 300, height: 400, alignment: .top)
+            Button {
+                sendDataStringToMiniApp(textString: $textToSend.wrappedValue)
+            } label: {
+                Text("Send Data to MiniApp")
+            }
+        }
+        .navigationTitle(pageName)
+        .trackPage(pageName: pageName)
+    }
+
+    func sendDataStringToMiniApp(textString str: String) {
+        for (_, handler) in viewModel.store.handlersListDict {
+            handler.sendJsonToMiniApp?(str)
+        }
```

Comment:
Just want to mention here as we discussed this works great for now but I think for a later state it would better fit the declarative approach to work with notifications or a different caching mechanism since there is a chance that the `MiniAppViewHandler` could be deallocated at random times

Reply:
I have not seen any issues with this yet. Seem to be working as intended.